src/auto/functions.rs

```diff
@@ -569,111 +573,193 @@ pub fn unix_is_system_fs_type(fs_type: &str) -> bool {
+    unsafe {
+        from_glib_full(gio_sys::g_unix_mount_guess_icon(
+            mount_entry.to_glib_none_mut().0,
+        ))
+    }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_guess_name(mount_entry: /*Ignored*/&mut UnixMountEntry) -> Option<GString> {
-//    unsafe { TODO: call gio_sys:g_unix_mount_guess_name() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_guess_name(mount_entry: &mut UnixMountEntry) -> Option<GString> {
+    unsafe {
+        from_glib_full(gio_sys::g_unix_mount_guess_name(
+            mount_entry.to_glib_none_mut().0,
+        ))
+    }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_guess_should_display(mount_entry: /*Ignored*/&mut UnixMountEntry) -> bool {
-//    unsafe { TODO: call gio_sys:g_unix_mount_guess_should_display() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_guess_should_display(mount_entry: &mut UnixMountEntry) -> bool {
+    unsafe {
+        from_glib(gio_sys::g_unix_mount_guess_should_display(
+            mount_entry.to_glib_none_mut().0,
+        ))
+    }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_guess_symbolic_icon(mount_entry: /*Ignored*/&mut UnixMountEntry) -> Option<Icon> {
-//    unsafe { TODO: call gio_sys:g_unix_mount_guess_symbolic_icon() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_guess_symbolic_icon(mount_entry: &mut UnixMountEntry) -> Option<Icon> {
+    unsafe {
+        from_glib_full(gio_sys::g_unix_mount_guess_symbolic_icon(
+            mount_entry.to_glib_none_mut().0,
+        ))
+    }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_is_readonly(mount_entry: /*Ignored*/&mut UnixMountEntry) -> bool {
-//    unsafe { TODO: call gio_sys:g_unix_mount_is_readonly() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_is_readonly(mount_entry: &mut UnixMountEntry) -> bool {
+    unsafe {
+        from_glib(gio_sys::g_unix_mount_is_readonly(
+            mount_entry.to_glib_none_mut().0,
+        ))
+    }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_is_system_internal(mount_entry: /*Ignored*/&mut UnixMountEntry) -> bool {
-//    unsafe { TODO: call gio_sys:g_unix_mount_is_system_internal() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_is_system_internal(mount_entry: &mut UnixMountEntry) -> bool {
+    unsafe {
+        from_glib(gio_sys::g_unix_mount_is_system_internal(
+            mount_entry.to_glib_none_mut().0,
+        ))
+    }
+}
 
 #[cfg(any(unix, feature = "dox"))]
 pub fn unix_mount_points_changed_since(time: u64) -> bool {
     unsafe { from_glib(gio_sys::g_unix_mount_points_changed_since(time)) }
 }
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_points_get() -> (/*Ignored*/Vec<UnixMountPoint>, u64) {
-//    unsafe { TODO: call gio_sys:g_unix_mount_points_get() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_points_get() -> (Vec<UnixMountPoint>, u64) {
```

Comment:
Similarly all these new functions should be part of the corresponding type's `impl`.